releng/org.eclipse.n4js.targetplatform/org.eclipse.n4js.targetplatform.target

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde version="3.8"?>
-<target name="Generated target-platform N4JS:N4JS_Neon.2.GH-18" sequenceNumber="125">
+<target name="Generated target-platform N4JS:N4JS_Neon.2.GH-18" sequenceNumber="127">
```

Comment:
the next number of 125 is 126? But this does not matter :D

Reply:
it is auto-generated number. Since I was generating targetplatform more than once, it happens to be incremented by more than one.